data/pidgin/cpsAssets/23248703.json

```diff
@@ -209,31 +209,31 @@
               "type": "listItem"
             },
             {
-              "text": "<link><caption>this is the link text</caption><altText>this is the alt text </altText><url href=\"https://bbc.com/pidgin\" platform=\"highweb\"/><url href=\"https://bbc.com/pidgin\" platform=\"enhancedmobile\"/></link>",
+              "text": "<link><caption>this is the link text</caption><altText>this is the alt text </altText><url href=\"https://www.bbc.com/pidgin\" platform=\"highweb\"/><url href=\"https://www.bbc.com/pidgin\" platform=\"enhancedmobile\"/></link>",
```

Comment:
Can you write a psammead issue to support `https://bbc.com` as internal please? Likely need to check `/` or `.` before `'bbc'`

Reply:
Done: https://github.com/bbc/psammead/issues/2496